Prepare Getting Started with Popup Guide (#2137)
DevExpress/devextreme-documentation: concepts/05 UI Components/Popup/00 Getting Started with Popup/05 Define the Title.md

Set the [showTitle](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) property to **true** to display popup's title bar and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to define its text: 

---
##### jQuery

    <!-- tab: index.js -->
    $(function () {
        $("#popup").dxPopup({
            // ...
            showTitle: true,
            title: "Information",
        });
        // ...
    });
 
##### Angular

    <!-- tab: app.component.html -->
    <dx-popup
        [showTitle]="true"
        title="Information">
        <!-- ... -->
    </dx-popup>
    <!-- ... -->

##### Vue

    <!-- tab: App.vue -->
    <template>
        <div id="app-container">
            <DxPopup
                :show-title="true"
                title="Information">
                <!-- ... -->
            </DxPopup>
            <!-- ... -->
        </div>
    </template>

    <script>
    // ...
    </script>

##### React

    <!-- tab: App.js -->
    // ...
    const App = () =>  {
        // ...
        return (
            <div className="App">
                <Popup
                    showTitle={true}

>                     title="Information"/>
>                 // ...

Reviewer: ```suggestion
                    title="Information"
                />
                {/* ... */}
```

> Set the [showTitle](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) property to **true** to display popup's title bar and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to define its text: 

Reviewer 2: ```suggestion
Set the [showTitle](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) property to **true** to display the Popup's title bar and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to define its text: 
```

> Set the [showTitle](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) property to **true** to display popup's title and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to define its content: 

Reviewer: ```suggestion
Set the [showTitle](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) property to **true** to display popup's title bar and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to define its text: 
```

---
##### jQuery

    <!-- tab: index.js -->
    $(function () {
        $("#popup").dxPopup({
            // ...
            showTitle: true,
            title: "Information",
        });
        // ...
    });

>     // ...

Reviewer: ```suggestion
```

 
##### Angular

    <!-- tab: app.component.html -->

>     // ...

Reviewer: ```suggestion
```

    <dx-popup
        [showTitle]="true"
        title="Information"
    >
        <div *dxTemplate="let data of 'content'">
            // ...

>         </div>

Reviewer: ```suggestion
        title="Information">
        <!-- ... -->
```

    </dx-popup>

    <!-- tab: app.component.ts -->
    import { Component } from '@angular/core';

    @Component({
        selector: 'app-root',
        templateUrl: './app.component.html',
        styleUrls: ['./app.component.css']
    })
    export class AppComponent {
        // ...
    }

> 

Reviewer: ```suggestion
```

##### Vue

    <!-- tab: App.vue -->
    <template>
        <div id="app-container">

>             // ...

Reviewer: ```suggestion
```

            <DxPopup
                :show-title="true"
                title="Information"         
            >
                <template #content>
                    // ...

>                 </template>            

Reviewer: ```suggestion
                title="Information">
                <!-- ... -->
```

            </DxPopup>
        </div>
    </template>

    <script>
    // ...

    export default {
        // ...

>     }

Reviewer: ```suggestion
```

    </script>

##### React

    <!-- tab: App.js -->
    import React, { useState } from 'react';

    import 'devextreme/dist/css/dx.common.css';
    import 'devextreme/dist/css/dx.light.css';
    
    // ...

> 

Reviewer: ```suggestion
    // ...
```

    function App() {
        // ...
        return (
            <div className="App">

>                 // ...

Reviewer: ```suggestion
```

                <Popup
                    showTitle={true}
                    title="Information"              
                >
                    // ...

>                 </Popup>

Reviewer: ```suggestion
                    title="Information"              
                />
```
